Decred Wallet/Features/Transactions/TransactionsViewController.swift

```diff
@@ -54,6 +54,7 @@ class TransactionsViewController: UIViewController {
         self.txTableView.addSubview(self.refreshControl)
         self.txTableView.hideEmptyAndExtraRows()
         self.txTableView.registerCellNib(TransactionTableViewCell.self)
+        self.txTableView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 56, right: 0)
```

Comment:
<img src="https://user-images.githubusercontent.com/19960200/120068160-2cc9dc80-c077-11eb-89ee-299eba48b623.jpg" width="300">
The transaction list stays directly above the floating buttons. There should be about 16-24px of space between them.